### PR TITLE
Fix README validation

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] In Progress
- [ ] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/48091

## Description
Added a condition to make sure the REAdME file exist before trying to open it

## Screenshots

Before

validateReadme

After

Screen Shot 2022-04-26 at 16 20 10

## Must have
- [x] Tests
- [x] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 1]
```diff
@@ -1509,14 +1509,18 @@ def verify_yml_commands_match_readme(self, is_modified=False):
...
         """
         if not is_modified:
             return True
         yml_commands_list = extract_none_deprecated_command_names_from_yml(self.current_file)
         is_valid = True
         dir_path = os.path.dirname(self.file_path)
         readme_path = os.path.join(dir_path, 'README.md')
+        if not os.path.exists(readme_path):
+            is_valid = False
+            return is_valid
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/integration.py:1522]:
```suggestion
        readme_path = Path(self.file_path).parent / 'README.md'
        if not readme_path.exists():
            return False
```

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 2]
```diff
@@ -1509,14 +1509,18 @@ def verify_yml_commands_match_readme(self, is_modified=False):
...
+
         with open(readme_path, 'r') as readme:
             readme_content = readme.read()
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/integration.py:1525]:
```suggestion
        readme_content = readme_path.read_text()
```

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 3]
```diff
@@ -1509,14 +1509,18 @@ def verify_yml_commands_match_readme(self, is_modified=False):
             is_modified (bool): Wether the given files are modified or not.
 
         Return:
-            bool: True if all commands appear in the readme, and False if it doesn't.
+            bool: True if all commands appear in the readme, and False if it doesn't or missing README.
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/integration.py:1512]:
```suggestion
            bool: True if all commands are documented in the README, False if there's no README file in the expected path, or any of the commands is missing.
```

**Review by dorschw — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,7 @@
 * Added time measurements to **lint**.
 * Added the flag '-d', '--dependency' to **find-dependencies** command to get the content items that cause the dependencies between two packs.
 * Fixed an issue where **doc-review** failed to recognize script names, in scripts using the old file structure.
+* Fixed an issue where **validate** failed when trying to read README.md file when the file does not exist.
```
Comment (dorschw) [inline CHANGELOG.md:10]:
```suggestion
* Fixed an issue where **validate** failed on nonexistent `README.md` files.
```

File: demisto_sdk/commands/common/hook_validations/integration.py (hunk 3 above)
Comment (barchen1) [reply demisto_sdk/commands/common/hook_validations/integration.py:1512]: done

**Pull request merged**